container-engine-lib/lib/backend_interface/objects/service/service_config_test.go

```diff
@@ -2,11 +2,11 @@ package service
 
 import (
 	"encoding/json"
-	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/image_download_mode"
-	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/image_registry_spec"
 	"testing"
 	"time"
 
+  "github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/image_download_mode"
```

Comment:
believe this is causing gofmt CI test to fail issue, and indent should fix then all tests should be good !

Reply:
should be fixed by [chore: go fmt](https://github.com/kurtosis-tech/kurtosis/pull/2257/commits/8382af06b550fdeee09d040b9cb73cf63b5d3ae2)